actions/class.RestItemComments.php

```
@@ -0,0 +1,123 @@
+use oat\tao\model\http\HttpJsonResponseTrait;
+use oat\taoItems\model\Comment\ItemCommentService;
+
+/**
+ * Item comments REST API (FR1 M1).
+ *
+ * Routes:
+ * - GET  /taoItems/RestItemComments/index?itemUri=
+ * - POST /taoItems/RestItemComments/index  (itemUri, body)
+ * - GET  /taoItems/RestItemComments/count?itemUri=
+ */
+class taoItems_actions_RestItemComments extends tao_actions_CommonModule
```

PHPStan (2.2.6)

[error] Class taoItems_actions_RestItemComments extends unknown class tao_actions_CommonModule.
Learn more at https://phpstan.org/user-guide/discovering-symbols

(class.notFound)